codes/quantum/oscillators/gkp.yml

```diff
@@ -3,25 +3,100 @@
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
 code_id: gkp
 physical: oscillators
-logical: qudits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'GKP code'
+
+# First reference with the code (optional).
+introduced: '\cite{doi:10.1103/PhysRevA.64.012310}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'It is a continuous-variable code that encode a logical qubit into a harmonic oscillator. Its stabilizer is displacement operators \(\hat{S}_q(2\alpha)=e^{-2i\alpha \hat{p}}\) and \(\hat{S}_p(2\beta)=e^{2i\beta \hat{q}}\). To ensure \(\hat{S}_q(2\alpha)\) and \(\hat{S}_p(2\beta)\) generate a stabilizer group which is Abelian, there is another constraint that \(\alpha\beta=2k\pi\) where \(k\) is an integer. Its codewords are equal weight superpositions of coherent states. When \(\alpha=\beta=\sqrt{\pi}\), \(|0_L\rangle=\sum_{j=-\infty}^\infty |2j\sqrt{\pi}\rangle_q\), \(|1_L\rangle=\sum_{j=-\infty}^\infty(2j+1)\sqrt{\pi}\rangle_q\). The codewords form a square lattice in phase space with spatial period \(2\sqrt{\pi}\).'
+
+protection: 'Protect against displacement errors and photon loss errors, but very sensitive to dephasing errors. Ref. \cite{doi:10.1103/PRXQuantum.2.020101}.'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Encode a 2-dimensional logical Hilbert space into an infinite dimensional Hilbert space. For stabilizer \(\hat{S}_q(2\alpha),\hat{S}_p(2\beta)\), it can correct displacement error up to \(\frac{\alpha}{2}\) at \(q\)-direction and \(\frac{\beta}{2}\) at \(p\)-direction.'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'The exact GKP state cannot be prepared.'
+    - 'Approximate GKP states can be prepared experimentally, by putting the GKP lattice inside a Guassian envelope \cite{doi:10.1038/s41586-020-2603-3}.'
+
+  #transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  general_gates:
+    - 'Pauli gates'
```

Comment:
Mention https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.123.200502

Reply:
Add part about how Clifford logical gates can be performed with Gaussian operations (from the original GKP paper).